bw2io/importers/__init__.py

```diff
@@ -13,3 +13,27 @@
 from .exiobase3_monetary import Exiobase3MonetaryImporter
 from .simapro_csv import SimaProCSVImporter
 from .simapro_lcia_csv import SimaProLCIACSVImporter
+
+"""
+This module provides classes for importing Life Cycle Impact Assessment (LCIA) data
+from different file formats. The following importers are provided:
+
+- EcoinventLCIAImporter
```

Comment:
Note to self: does it make sense to list these here? On the documentation website, they would be listed below anyway?

Reply:
@michaelweinold Do I need to fix something here?

Reply:
I think it makes sense to remove this here. On the website, the importers will be listed anyway. That way we don't have to hard-code these in the docstrings.

Reply:
@michaelweinold Resolved with https://github.com/brightway-lca/brightway2-io/pull/166/commits/a3d8ef4e450c5c2718797903933c3ed5ae9f1c5f